### DISCUSSION TITLE
Updating Outdated Branches in Pull Requests

### DISCUSSION THREAD

**Discussion by MohamadSalman11** [twbs/bootstrap#40912] [Q&A] [answered]
When I make a pull request and the main branch gets updated while my pull request hasn't been merged, I see that my branch is outdated, and I have the option to click a button to update it. Should I update the branch every time it's outdated, or will a team member update it before merging the pull request?

**Comment by julien-deramond** [comment, marked as answer]
You don't need to update the branch every time it's outdated, as long as there are no conflicts (the interface will indicate this). The core team will update it during the review if necessary :)

**Reply by MohamadSalman11** [reply]
Thank you very much.
